vembrane/modules/filter.py

```diff
@@ -116,15 +119,23 @@ def add_subcommmand(subparsers):
         "given in the VCF header. "
         "Example: `--overwrite-number-format DP=2`",
     )
+    parser.add_argument(
+        "--backend",
+        "-b",
+        default="pysam",
+        type=Backend.from_string,
+        choices=[Backend.pysam, Backend.cyvcf2],
+        help="Set the backend library.",
+    )
```

Comment:
I'd love to get rid of the code duplication in the different modules somehow.

Reply:
I moved this code to a common function in common.py